Mob/mob.gd

```diff
@@ -0,0 +1,11 @@
+extends CharacterBody2D
+
+var player 
+
+func _ready():
+	player = get_node("/root/Game/Player")
```

Comment:
This is problematic. Hardcoding like this will be a problem. If the player is elsewhere, or doesn't even exist then the code below will also crash the game. We'll need to discuss how we're going to deal with this in a meeting.

Reply:
i think adding the player scene in a group called "Player" or adding it as a autoload/singleton would work